src/Components/Rockets.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRocketsData, reserveRocket, unreserveRocket } from '../redux/rockets/rocketsSlice';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+
+  useEffect(() => {
+    dispatch(fetchRocketsData());
+  }, [dispatch]);
```

Comment:
- [ ] Kindly make sure the fetched data are not being added to the store whenever a user changes view / uses navigations.

    - ### APP BEHAVIOUR
    
   ![chrome-capture-2023-4-25 (2)](https://github.com/tomdan-ai/Travalers-space-hub/assets/87197412/6a5f857a-7aa8-46a0-a146-dccd8032a0fd)

 
   


  _**Hint: Make sure to dispatch the `fetchRocketsData()` action only when the `missions`  array in the redux store is empty**_